3rdparty/openjpeg/openjp2/ht_dec.c

```diff
@@ -55,6 +55,10 @@
 #define OPJ_COMPILER_GNUC
 #endif
 
+#if defined(OPJ_COMPILER_MSVC) && defined(_M_ARM64)
+#include <arm64_neon.h>
```

Comment:
> 3rdparty

We usually don't modify 3rdparty code. Because it complicates maintenance and such changes could be lost after 3rdparty library upgrade.
Is where similar patch / commit in the upstream repository of 3rdparty library?

Reply:
This actually follows onto the fix made in #24054.  I don't know if that's been made upstream yet.